src/analysis/history_utils.hpp

```diff
@@ -0,0 +1,56 @@
+// © 2022. Triad National Security, LLC. All rights reserved.  This
+// program was produced under U.S. Government contract
+// 89233218CNA000001 for Los Alamos National Laboratory (LANL), which
+// is operated by Triad National Security, LLC for the U.S.
+// Department of Energy/National Nuclear Security Administration. All
+// rights in the program are reserved by Triad National Security, LLC,
+// and the U.S. Department of Energy/National Nuclear Security
+// Administration. The Government is granted for itself and others
+// acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+// license in this material to reproduce, prepare derivative works,
+// distribute copies to the public, perform publicly and display
+// publicly, and to permit others to do so.
+
+#ifndef ANALYSIS_HISTORY_UTILS_HPP_
+#define ANALYSIS_HISTORY_UTILS_HPP_
+
+#include <string>
+#include <vector>
+
+#include "geometry/geometry.hpp"
+#include "geometry/geometry_utils.hpp"
+#include "phoebus_utils/relativity_utils.hpp"
+#include "phoebus_utils/variables.hpp"
+#include <kokkos_abstraction.hpp>
+#include <parthenon/package.hpp>
+#include <utils/error_checking.hpp>
```

Comment:
It doesn't matter much, but I like to space-separate teh parthenon vs phoebus headers. And also, due to lore, I like to order it:
- stdlib headers
- dependencies such as parthenon
- phoebus

so in this case that would look like:
```suggestion
#include <kokkos_abstraction.hpp>
#include <parthenon/package.hpp>
#include <utils/error_checking.hpp>

#include "geometry/geometry.hpp"
#include "geometry/geometry_utils.hpp"
#include "phoebus_utils/relativity_utils.hpp"
#include "phoebus_utils/variables.hpp"
```